website/docs/r/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,92 @@
+* `custom_service_type` - (Optional) Custom Service type. A custom service type used when this EPG is use as custom service EPG with public or private access. As an e.g. this is used to provide the service tag for an Azure service. Type: string
+* `deployment_type` - (Optional) cloud service deployment type.deploymentType refers to the type of deployment of the service. It could be a PaaS service, a PaaS service deployed in the managed VNET/VPC, a SaaS service consumed or a SaaS service offered. Allowed values are "CloudNative", "CloudNativeManaged", "Third-party","Third-partyManaged", "Unknown", and default value is "Unknown". Type: String.
+* `flood_on_encap` - (Optional) Handling of L2 Multicast/Broadcast and Link-Layer traffic at EPG level. Control at EPG level if the traffic L2 Multicast/Broadcast and Link Local Layer should be flooded only on ENCAP or based on bridg-domain settings. Allowed values are "disabled", "enabled", and default value is "disabled". Type: String.
+* `label_match_criteria` - (Optional) Provider Label Match Criteria. Allowed values are "All", "AtleastOne", "AtmostOne", "None", and default value is "AtleastOne". Type: String.
+* `preferred_group_member` - (Optional) Represents parameter used to determine if the Cloud Service EPG is part of a group that does not a contract for communication. Allowed values are "exclude", "include", and default value is "exclude". Type: String.
+* `prio` - (Optional) The QoS priority class identifier. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+* `cloud_service_epg_type` - (Optional) The specific service type of the object or component. Allowed values are "Azure-ADDS", "Azure-AksCluster", "Azure-ApiManagement", "Azure-ContainerRegistry", "Azure-CosmosDB", "Azure-Databricks", "Azure-KeyVault", "Azure-Redis", "Azure-SqlServer", "Azure-Storage", "Azure-StorageBlob", "Azure-StorageFile", "Azure-StorageQueue", "Azure-StorageTable", "Custom", "Unknown", and default value is "Unknown". Type: String.
+* `relation_cloud_rs_cloud_epg_ctx` - (Optional) Represents the relation to a Relationship to the VRF of belonging (class fvCtx). Type: String.
+* `relation_fv_rs_cons` - (Optional) A block representing the relation to a Contract Consumer (class vzBrCP). The Consumer contract profile information. Type: Block.
+  * `prio` - (Optional) The system class determines the quality of service and priority for the consumer traffic. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+  * `target_dn` - (Required) The distinguished name of the target. Type: String
+* `relation_fv_rs_cons_if` - (Optional) A block representing the relation to a Contract Interface (class vzCPIf). A contract for which the EPG will be a consumer. Type: Block.
```

Comment:
-> A block representing the relation to a Contract Interface (class vzCPIf) for which the EPG will be a consumer. Type: Block.